src/test/java/essenmakanan/recipe/EditRecipeTest.java

```diff
@@ -66,4 +68,22 @@ public void editRecipe_invalidInput_exceptionThrown() {
             recipes.editRecipe(recipeToEdit, editDetails);
         });
     }
+
+
+    /*@Test
+    public void editRecipeNameAndSteps_validInput_editSuccess() {
+        // recipe title with multiple word
+        String[] editDetails = {"edit", "n/white bread", "s/1,Prepare the dough"};
+        try {
+            recipes.editRecipe(recipeToEdit, editDetails);
+        } catch (EssenFormatException e) {
+            e.handleException();
+        }
+
+        String newStep = recipeToEdit.getRecipeSteps().getStepByIndex(0).getDescription();
+        assertEquals("Prepare the dough", newStep);
+        assertEquals("Breads", recipeToEdit.getTitle());
+    }*/
+
```

Comment:
Is this test going to be used?

Reply:
yes, i uncommented it in the next iteration